ee/desktop/menu/menu_icon.go

```diff
@@ -0,0 +1,64 @@
+package menu
+
+import (
+	"runtime"
+
+	"github.com/go-kit/kit/log/level"
+	"github.com/kolide/launcher/ee/ui/assets"
+)
+
+func (m *menu) getIcon(icon menuIcon) []byte {
+	switch icon {
+	case UnregisteredIcon:
+		return chooseIcon(
+			assets.MenubarTranslucentDarkmodeIco,
+			assets.MenubarTranslucentDarkmodePng,
+			assets.MenubarTranslucentLightmodeIco,
+			assets.MenubarTranslucentLightmodePng,
+		)
+	case DefaultIcon:
+		return chooseIcon(
+			assets.MenubarDefaultDarkmodeIco,
+			assets.MenubarDefaultDarkmodePng,
+			assets.MenubarDefaultLightmodeIco,
+			assets.MenubarDefaultLightmodePng,
+		)
+	case BlockingSoonIcon:
+		return chooseIcon(
+			assets.MenubarTriangleExclamationDarkmodeIco,
+			assets.MenubarTriangleExclamationDarkmodePng,
+			assets.MenubarTriangleExclamationLightmodeIco,
+			assets.MenubarTriangleExclamationLightmodePng,
+		)
+	case BlockedIcon:
+		return chooseIcon(
+			assets.MenubarCircleXDarkmodeIco,
+			assets.MenubarCircleXDarkmodePng,
+			assets.MenubarCircleXLightmodeIco,
+			assets.MenubarCircleXLightmodePng,
+		)
+	default:
+		level.Debug(m.logger).Log(
+			"msg", "invalid icon",
+			"icon", icon)
+		return nil
+	}
+}
+
+func chooseIcon(darkIco, darkPng, lightIco, lightPng []byte) []byte {
```

Comment:
Should we bring in monochrome as choice?

Reply:
The monochrome ones are the alpha channel only ones, if we didn't have color. I'm not sure we'll ever chose them, but I feel like the hooks should be there